sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -257,10 +270,15 @@ public String getEntityPath() {
         if (lockToken instanceof ServiceBusReceivedMessage) {
             return complete(lockToken, ((ServiceBusReceivedMessage) lockToken).getSessionId());
         } else {
-            return complete(lockToken, null);
+            String sessionId = null;
```

Comment:
Why create a variable for this?

Reply:
There are overloads for complete() which makes this call ambiguous for compiler. This this was needed, But I am replacing this with direct call to `updateDisposition`